openedx/features/enterprise_support/api.py

```diff
@@ -450,7 +450,7 @@ def enterprise_customer_for_request(request):
     Check all the context clues of the request to determine if
     the request being made is tied to a particular EnterpriseCustomer.
     """
-    if 'enterprise_customer' in request.session:
+    if 'enterprise_customer' in request.session and request.session['enterprise_customer']:
```

Comment:
had to add this extra check because this if statement was evaluating to true when `request.session['enterprise_customer']` was `None`

Reply:
Maybe just change the whole conditional to `request.session.get('enterprise_customer')` since `None` is falsey?

Reply:
FYI I changed to your suggested statement but it broke a test so I reverted it